### PR TITLE
Jekyll migration - initial commit

### PR DESCRIPTION
**Pull request by indraraj**
Initial commit

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@Naros, can you take a look at this one? Note we converged on Jekyll in the end, as it's the platform that's "supported" by the design team; I.e. if we get a fresh style from them at some point, they'd deliver this as Jekyll templates, but not Hugo.

**Comment by gunnarmorling** [comment]
@indraraj, also added you as committers to this repo, i.e. you can directly push to upstream/debezium.github.io

**Comment by gunnarmorling** [comment]
@indraraj, thanks a lot! Could you perhaps describe the current state? What's working, what's not? And how to build and preview the thing?

**Comment by indraraj** [comment]
Hey, @gunnarmorling  thanks for providing access to the repo.
Regarding the steps to build and running locally, you can find these details on the readme page, and For what's working it's basically in the same state that we demoed in the last call(Home page, FAQ, Community).

File: ANTORA.md [hunk 1]
```diff
@@ -1,82 +0,0 @@
-# Antora Integration
```
Comment (gunnarmorling) [inline ANTORA.md:1]: Why is this (and other similar) files deleted? We'd definitely want to keep those.

File: CNAME [hunk 2]
```diff
@@ -1 +0,0 @@
-debezium.io
```
Comment (gunnarmorling) [inline CNAME:1]: Is this going elsewhere? It's definitely needed in the output published to GH Pages.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,364 +0,0 @@
-## Contribute to Debezium's website
```
Comment (gunnarmorling) [inline CONTRIBUTING.md:1]: Same here?

File: _data/releases/0.1/0.1.0.Final.yml [hunk 4]
```diff
@@ -1,3 +0,0 @@
-date: 2016-03-17
```
Comment (gunnarmorling) [inline _data/releases/0.1/0.1.0.Final.yml:1]: The data files should remain and drive the releases page.

**Review by gunnarmorling — COMMENTED** [review]
@indraraj didn't dive too much, but it seems there are quite few files deleted which should remain. Are you planning to bring those back?

**Comment by indraraj** [comment]
yes @gunnarmorling , as its WIP and we will bring those back. As mention, this was just the initial commit.

As of right now, we have been working from @uidoyen repo and we wanted to move that here.

**Comment by mdrillin** [comment]
I pulled down and tested - worked ok for me.  I think this is fine for initial commit for the migration branch.

**Comment by gunnarmorling** [comment]
Ok then. Could you only rework the commit with an email associated to your GH account (if done right, you'll see your profile picture next to the commit instead of the grey octocat)? Once that's done feel free to merge this PR yourself into upstream/website-migration, so to test whether the commit rights are correct. Thx!

**Comment by gunnarmorling** [comment]
Thx, @indraraj!

**Pull request merged**